Web_Development/FrontEnd/HTML Links.md

```diff
@@ -0,0 +1,27 @@
+## HTML Links
+
+    • To link an external documents to the current document we use <link> HTML element. It is also denoted with hyperlinks. It can be written    
+      inside the <body> or inside the <head>.
+
+            example :
+            <link rel="stylesheet" href="">
+
+    • "href" attribute is used to provide the path or way to the stylesheet(mostly it is URL).
```

Comment:
Use correct way of using unordered list in Markdown.
Use inline code for "href": Use `href` not "href".